pom.xml

```diff
@@ -85,21 +85,18 @@
     <google.api-client-libraries.version>2.0.0</google.api-client-libraries.version>
     <google.api-iamcredentials.version>v1-rev20211203-${google.api-client-libraries.version}</google.api-iamcredentials.version>
     <google.api-storage.version>v1-rev20220705-${google.api-client-libraries.version}</google.api-storage.version>
-    <google.api.grpc.proto-google-iam-v1.version>1.6.23</google.api.grpc.proto-google-iam-v1.version>
     <google.auth.version>1.14.0</google.auth.version>
     <google.auto-value.version>1.10.1</google.auto-value.version>
     <google.cloud-core.verion>2.9.0</google.cloud-core.verion>
-    <google.cloud-storage.bom.version>2.25.0</google.cloud-storage.bom.version>
+    <!-- Library bom for Java-Storage -->
+    <google.cloud-storage.library.bom.version>26.21.0</google.cloud-storage.library.bom.version>
```

Comment:
Libraries-bom will apply to all cloud clients, gax, guava, grpc, http client, etc. 

I'd probably remove storage from the property name.

Reply:
updated.